concourse/pipeline.yml

```diff
@@ -136,6 +139,7 @@ jobs:
         params:
           CF_SPACE: production
           INSTANCES: 20
+          WORKER_INSTANCES: 1
```

Comment:
Would be cool to keep an eye on this, since this application is fairly high traffic, whether we'd need more worker instances..

There's a card on our planning board to capture sidekiq stats in our prometheus metrics, which should help.

Reply:
Yeah, I'll keep an eye on this for the next few days